---
title: Real-time genomics to track COVID-19 post-elimination border incursions in Aotearoa New Zealand
authors:
  - Jordan Douglas, PhD
  - Jemma L. Geoghegan, PhD
  - James Hadfield, PhD
  - Remco Bouckaert, PhD
  - Matthew Storey, MSc
  - Xiaoyun Ren, PhD
  - Joep de Ligt, PhD
  - Nigel French, PhD
  - David Welch, PhD
affiliations: "Correspondence: jordan.douglas@auckland.ac.nz; The University of Auckland, Private Bag 92019, Auckland 1142, New Zealand"
dataset: "http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions"
date: August 2021
abstract: "
This is an [interactive Nextstrain narrative](https://docs.nextstrain.org/en/latest/guides/communicate/narratives-intro.html) which is a companion to the scientific paper [Douglas et al., EID, (2021)](https://wwwnc.cdc.gov/eid/article/27/9/21-1097_article)). We recount the thirteen community outbreaks from when the virus was successfully eliminated in May 2020 to June 2021, and reflect on how genomics played a vital role in understanding, managing and containing these incursions. 

Of the thirteen COVID-19 community outbreaks we look at, two led to stay-at-home orders being issued by health officials. All outbreaks originated at the border; via returnees, airline workers, and cargo vessels. With a public health system informed by real-time viral genome sequencing, which typically had complete genomes within 12 hours after a community-based positive COVID-19 test, outbreaks were well-contained. These incursions resulted in a total of 225 community cases,  which unfortunately led to three deaths. Real-time genomics was a vital piece of information for establishing links between cases when epidemiological data could not be obtained or was inconclusive, genomics also assisted in the identification of different origins for concurrent outbreaks. 
"
---


# [Compassionate Exemption Cases](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Compassionate)

After 24 days without any recorded cases in the community or at the border, and one week after the whole of Aotearoa had moved down to Alert Level 1, two cases were identified in the community. They had arrived on 7 June 2020 and were granted a compassionate exemption to exit a managed isolation and quarantine (MIQ) facility early in order to attend a funeral on 13 June. The conditions of the exemption required them to self-isolate as far as possible while travelling and to get tested.

On June 16, they both tested positive. Within one day, complete viral genome sequencing confirmed that the two cases shared a single origin. These genomes are coloured in green on the phylogenetic tree highlighted on the right.

Although there were no secondary infections, concern for public health saw the New Zealand Defence Force put in charge of managing MIQ facilities, no returnees being allowed to leave MIQ without first returning a negative test, and the end of most compassionate exemptions from MIQ requirements.


# [Auckland August Lockdown](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Auckland%20August&label=internal:mar27)

On 11 August 2020, four cases of COVID-19 were found among workers at an Auckland cold storage facility after a 102 day period with no recorded community transmission. The city was put into an immediate lockdown (Alert Level 3), and lower level restrictions were introduced for the rest of the country (Alert Level 2). Elevated restrictions remained until 7 October as the cluster grew to a total of 179 cases, including 3 deaths. This was the largest COVID-19 cluster in New Zealand to date.

The genomic data indicated a single origin which provided confidence to public health officials that it was a single outbreak despite [several cases being found that had no clear epidemiological links with other cases](https://www.health.govt.nz/news-media/mediareleases/9-new-cases-covid-19-2). The cluster [included a healthcare worker who was infected during their work at an MIQ facility where community cases were sent to quarantine.](https://www.health.govt.nz/news-media/mediareleases/1-new-case-covid-19-18) While the index cases worked at a cold chain supply facility linked to the border, [the source of the outbreak was never established](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8084492/).

Complete genomes are available for 87% of cases in this outbreak which we believe makes it one of the most comprehensively sampled large COVID-19 outbreaks.

# [Rydges MIQ Facility](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Auckland%20August,Rydges&label=internal:mar27)

During the Auckland August outbreak, there was an unusual case of a maintenance worker at the Rydges MIQ facility (here coloured in blue) without an epidemiological link to the ongoing community outbreak. Sequencing confirmed that the source of infection was not related to the main community cluster, but rather to an overseas returnee under managed isolation at the Rydges MIQ facility.

A follow-up investigation suggested that transmission likely occurred when the two cases [used the same elevator in close succession](https://www.health.govt.nz/news-media/media-releases/11-new-cases-covid-19-0) and was likely a case of airborne transmission.


# [Crowne Plaza MIQ Facility](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Auckland%20August,Rydges,Crowne%20Plaza&label=internal:mar27)

In September 2020 while the Auckland August cluster was still active, a returnee developed symptoms and tested positive in Auckland four days after ending their 14 day isolation period in the Christchurch Crowne Plaza MIQ facility. 

Genome sequencing revealed that this case (“Case G”) – and their two household contacts – were not linked to Auckland cluster. But they were linked to other returnees at the Crowne Plaza MIQ. Investigation revealed a complex chain of transmissions: Cases A and B were seated close to Case C on a repatriation flight from India. The virus then jumped from case C to case D in the MIQ facility via airborne transmission between a hotel room and its adjacent hallway. Case D is then thought to have been infectious on a domestic flight from Christchurch to Auckland where case G caught the virus. (Cases E and F were household contacts of others).

This powerful combination of genome sequencing and detailed epidemiological investigation enabled the identification of cryptic transmission events on flights, and airborne transmission between MIQ guests.

The genomic evidence here is complete with all sequences in the cluster available and the tree relatively well-resolved with four distinct genomes among the nine cases.

Full details of this outbreak are in [Eichler et al. (2021)](https://wwwnc.cdc.gov/eid/article/27/5/21-0514_article).


# [Multiple Outbreaks in the Sudima MIQ facility](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?c=lineage&f_outbreak=Sudima&label=internal:sudima_mrca)

235 international mariners arrived into New Zealand on a charter flight from Moscow on 16 October 2020 and began their self-isolation at the Sudima MIQ facility in Christchurch. In the ensuing period, 31 of the mariners tested positive for SARS-CoV-2 as did [two workers at the MIQ facility](https://www.health.govt.nz/news-media/media-releases/international-mariners-leave-managed-isolation). Genomic sequencing produced full genomes for 24 cases out of the 33 cases, showing three distinct phylogenetic clades. 

The outbreak is highlighted here and with colours showing their PANGO lineage.

We estimate the origin of the largest clade to be between 4 August and 9 October (95% credible interval), but the mariners arrived on 16 October, suggesting there were at least two separate introductions of this variant into the facility, so at least four introductions overall.

The two MIQ workers were infected with distinct variants indicating different returnee to worker transmission events. [Officials concluded](https://www.mbie.govt.nz/dmsdocument/12887-lessons-learned-review-international-mariners) protocols were followed but this had been insufficient to prevent transmission.

# [Sofrana Surville Cargo Vessel](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Sofrana%20Surville&label=internal:may18)

In mid October, a border worker tested positive as part of routine testing. After extensive surveillance testing, three other cases were identified; two were household contacts of the border worker and one was a mariner who had worked on board the same cargo vessel – the Sofrana Surville. Genomic testing confirmed that the four cases shared the same origin, and were part of a lineage that was novel to New Zealand, thus making residual community transmission an unlikely explanation. 

THe source was confirmed when crew members on board the [Sofrana Surville](https://www.health.govt.nz/news-media/media-releases/confirmed-cases-covid-19-sofrana-surville) were tested in Queensland. One tested positive and the sequence was reported to match the New Zealand cases.


# [Defence Force Cluster](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Defence%20Force&label=internal:may18)

After visiting a number of locations in the Auckland central business district, a New Zealand Defence Force member tested positive. Genome sequencing confirmed the infection had been acquired at the quarantine facility they worked at. Contact tracing identified three additional cases among the worker's close contacts in Wellington.

Surveillance testing identified [one community case](https://www.health.govt.nz/news-media/media-releases/3-new-cases-covid-19-12) in the Auckland central business district.  The only known possible epidemiological link to Defence worker was that the community case worked at a retail outlet about 50 metres from one of the locations the index case had visited. The community case was quickly sequenced and was genomically linked to the rest of the cluster, providing reassurance that widespread undetected community transmission was unlikely (NZMH, 2020d). The circumstances of the exact transmission event remains unknown. 

WIth the genomic link established, there was no need to change Alert Levels. However, the general public were asked to avoid the Auckland central business district where possible for three days.


# [Airline Crew cases 1 & 2](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Airline%20Crew%201,Airline%20Crew%202)

There have been two separate cases of airline crew testing positive for COVID-19 after international travel.

In the [first case](https://www.health.govt.nz/news-media/media-releases/1-new-case-covid-19-23), in December 2020, the crew member had returned from a flight from the United States of America. They were detected within 48 hours and there were no recorded secondary infections. Genome sequencing indicated they likely acquired their infection abroad.

The [second case](https://www.health.govt.nz/news-media/media-releases/confirmed-case-border) happened in March 2021 (during the Auckland Valentine’s outbreak). Here an airline crew member tested positive a week after their arrival from Japan. Their infection was likely acquired overseas and no secondary infections were identified.


# [Pullman MIQ Facility](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Pullman&label=internal:apr04)

A returnee tested positive for SARS-CoV-2 one week after completing managed isolation at the Pullman MIQ facility. Despite travelling extensively with a companion across the Northland region whilst infected with the [B.1.351, or Beta, lineage](https://www.nature.com/articles/s41586-021-03471-w), there were no secondary infections reported. 

Shortly thereafter, two additional cases with the same variant were found in the community, both of whom had also  [self-isolated at the Pullman](https://www.health.govt.nz/news-media/media-releases/3-new-cases-covid-19-border-and-update-border-related-cases-auckland). The outbreak was successfully limited to these four community cases who were genomically linked to a returnee under in isolation at the Pullman. 

The epidemiological link was thought to be the air circulation system at the MIQ facility. New arrivals at the MIQ facility were suspended and its air filtration systems were improved.

# [Auckland Valentine’s Lockdowns](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Auckland%20Valentine&label=outbreak:Auckland%20Valentine)

On the eve of Valentine’s day 2021, three household members in the community in Auckland tested positive. The genomes were indistinguishable from each other and of the highly transmissible [B.1.1.7, or Alpha, variant](https://www.nature.com/articles/s41586-021-03470-x). Although one case worked at an airline services company at the Airport, they had no obvious contact with people coming through the border. Auckland was immediately sent into its third Alert Level 3 lockdown for a period of three days, and wide scale surveillance testing commenced. Other cases  - not always with very strong epidemiological links – were found with closely related genomes but the outbreak was thought to be controlled and the region moved to a lower Alert Level. This was reversed as the fourth Alert Level 3 lockdown was imposed when a further community case was found that could not immediately be linked to the cluster.

Eventually, the  outbreak was restricted to four households and 15 cases, although epidemiological gaps suggest there may have been undetected cases. With genetic evidence showing four distinct genomes among 14 cases, the genomics were not informative about the likely original source of the outbreak, beyond confirming a single origin. The Alpa variant’s global overrepresentation presented further difficulties at pinpointing an overseas origin. The outbreak has not been traced back beyond the original three cases and its origin remains unknown.


# [Grand Millennium MIQ Facility](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Auckland%20Valentine,Grand%20Millennium&label=internal:sep26)
In late March 2021, a cleaner at the Grand Millennium MIQ facility tested positive. Despite being infected with the more transmissible Alpha variant, there were no known secondary infections in the community. Two weeks later, two further MIQ workers from the same facility also tested positive. All three cases were genomically linked back to a returnee isolating at the facility.

# [Auckland Airport](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Auckland%20Valentine,Grand%20Millennium,Auckland%20Airport&label=internal:sep26)

A worker at Auckland International Airport tested positive for the Alpha variant in April 2021. No onward transmission was detected. Genome sequencing linked the case to a recent returnee, and a followup investigation suggested that worker’s infection was obtained from cleaning the airplane the returnee had arrived on, and was [likely a case of airborne transmission](https://www.health.govt.nz/news-media/media-releases/no-new-community-cases-1-new-case-covid-19-managed-isolation).


# [A wide range of lineages](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?c=lineage)

Zooming out and colouring the tree by PANGO lineage, we can see that a [wide range of lineages have been imported into Aotearoa](https://www.nature.com/articles/s41467-020-20235-8) over the course of the pandemic. The fact that the four locally acquired outbreaks in early 2021 were all caused by variants of concern (VoC) – Pangolin lineages B.1.1.7 and B.1.351 – reflects the lineages that are arriving at the border, with 83 out of the 142 genomes from overseas returnees found from 1 January to 30 April 2021 being from those two lineages or other VoCs. However, from this limited data we cannot draw conclusions on the  transmissibility of these VoCs.


# [Final words](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?c=lineage)

New Zealand’s ability to rapidly generate SARS-CoV-2 genomes has greatly improved during the early months of the pandemic, to the point where genomes of newly diagnosed patients are now routinely available within 24 hours. Although the potential of the techniques described in our paper have been [well characterised in academia](https://www.nature.com/articles/nrg.2017.88), the pandemic has led to widespread adoption in New Zealand's public health system,  and the term “whole genome sequencing” becoming commonplace in public health announcements. 

Combined with epidemiological investigation, genomic data have increased public understanding of the outbreak and driven policy changes to minimise infection risks and disruption to society.

For more information please see the paper on which this narrative is based on [here](https://wwwnc.cdc.gov/eid/article/27/9/21-1097_article), or click "Explore the Data Yourself" (top right of screen) to interact with this tree.
