---
title: Real-time genomics to track COVID-19 post-elimination border incursions in Aotearoa New Zealand
authors:
  - Jordan Douglas, PhD
  - Jemma L. Geoghegan, PhD
  - James Hadfield, PhD
  - Remco Bouckaert, PhD
  - Matthew Storey, MSc
  - Xiaoyun Ren, PhD
  - Joep de Ligt, PhD
  - Nigel French, PhD
  - David Welch, PhD
affiliations: "Correspondence: jordan.douglas@auckland.ac.nz; The University of Auckland, Private Bag 92019, Auckland 1142, New Zealand"
dataset: "http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions"
date: June 2020
abstract: "
This is an [interactive Nextstrain narrative](https://docs.nextstrain.org/en/latest/guides/communicate/narratives-intro.html) which is a companion to the scientific paper [Douglas et al., medRxiv (2021)](https://www.medrxiv.org/content/10.1101/2021.05.13.21257194v1.full.pdf+html). We recount the thirteen community outbreaks since the virus was successfully eliminated in May 2020, and reflect on how genomics played a vital role in understanding, managing and containing these incursions.

There have been thirteen known COVID-19 community outbreaks in Aotearoa New Zealand since the virus was first eliminated in May 2020, two of which led to stay-at-home orders being issued by health officials. These outbreaks originated at the border; via isolating returnees, airline workers, and cargo vessels. With a public health system informed by real-time viral genome sequencing, which typically had complete genomes within 12 hours after a community-based positive COVID-19 test, outbreaks were well-contained. These incursions resulted in a total of 225 community cases, in which unfortunately led to three deaths. Real-time genomics was a vital piece of information for establishing links between cases when epidemiological data could not or was inconclusive, genomics also assisted in the identification of different origins for concurrent outbreaks. By combining epidemiological and contact tracing data we reconstruct the viral transmission history from genomic sequences. In the following pages we recount the thirteen community outbreaks and demonstrate how genomics played a vital role in containing them. 
"
---


# [Compassionate Exemption Cases](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Compassionate)

After 24 days without any recorded cases in the community or at the border, and one week after the whole of Aotearoa had moved down to Alert Level 1, two cases were found identified in the community. They had arrived on 7 June and were granted a compassionate exemption to exit managed isolation and quarantine (MIQ) facility earliery in order to attend a funeral on 13 June. The conditions of the exemption required them to self-isolate as far as possible while travelling and to get tested.

They both tested positive for COVID-19 on June 16. Within one day, complete viral genome sequencing confirmed that the two cases shared a single origin. These genomes are coloured in green hereon the phylogenetic tree highlighted on the right.

Although there were no secondary infections, concern for public health saw the New Zealand Defence Force put in charge of managing MIQ facilities, no returnees being allowed to leave MIQ without first returning a negative test, and the end of compassionate exemptions from most of the MIQ requirements.


# [Auckland August Lockdown](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Auckland%20August&label=internal:mar27)

On 11 August 2020, four cases of COVID-19 were found among workers at an Auckland cold storage facility ending after a 102 day period with no recorded community transmission. The city was sent put into an immediate lockdown (Alert Level 3), and lower level restrictions were introduced for the rest of the country (Alert Level 2). Elevated restrictions remained until 7 October as the cluster grew to a total of 179 cases, including 3 deaths. This was the largest COVID-19 cluster in New Zealand to date.

The genomic data indicated a single origin which provided confidence to public health officials that it was a single outbreak despite [several cases being found that had no clear epidemiological links with other cases](https://www.health.govt.nz/news-media/mediareleases/9-new-cases-covid-19-2). The cluster [included a healthcare worker who was infected during their work at an MIQ facility where community cases were sent to quarantine.](https://www.health.govt.nz/news-media/mediareleases/1-new-case-covid-19-18) While the index cases worked at a cold chain supply facility linked to the border, [the source of the outbreak was never established](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8084492/).

Complete genomes are available for 87% of cases in this outbreak which we believe makes it one of the most comprehensively sampled large COVID-19 outbreaks.

# [Rydges MIQ Facility](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Auckland%20August,Rydges&label=internal:mar27)

During the Auckland August outbreak, there was an unusual case of a maintenance worker at the Rydges MIQ facility (here coloured in blue) witohout no an known epidemiological link to the ongoing community outbreak. Sequencing confirmed that the source of infection was not related to the main community cluster, but rather to an overseas returnee under managed isolation at the Rydges MIQ facility.

A followup investigation suggested that transmission likely occurred when the two cases used the same elevator in close succession (NZMH, 2020f) and was likely a case of airborne transmission (van Rijn et al., 2020; Dbouk and Drikakis, 2021).

# [Crowne Plaza MIQ Facility](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Auckland%20August,Rydges,Crowne%20Plaza&label=internal:mar27)

In September 2020, a returnee developed symptoms and tested positive in Auckland four days after leaving the Crowne Plaza MIQ facility in Christchurch. Genomeic sequencing showed revealed that this case (“Case G”) – and their two household contacts – were not linked to the ongoing Auckland August cluster. Rather, they were linked to other returnees under managed isolation at the Crowne Plaza. Epidemiological investigations show the chain likely started with Cases A and B who were seated close to Case C on a repatriation flight from India. The virus then jumped from case C went on to infect case D in the MIQ facility via airborne transmission between a hotel room and its adjacent hallway. Case D is then thought to have infected been infectious G on a domestic flight from Christchurch to Auckland where case G caught the virus. This outbreak illustrates the power of genome sequencing – when coupled with detailed epidemiological investigations – for identifying cryptic transmission events such as those on flights or airborne transmission between MIQ guests. The genomic evidence here is complete with all sequences in the cluster available and the tree relatively well-resolved with four distinct genomes among its the nine cases.

The complete Crowne Plaza outbreak has been thoroughly analysed in Eichler et al. (2021) so we provide only brief detail here.

# [Multiple Outbreaks in the Sudima MIQ facility](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?c=lineage&f_outbreak=Sudima&label=internal:sudima_mrca)

235 international mariners arrived into New Zealand on a charter flight from Moscow on 16 October 2020 and began their self-isolation at the Sudima MIQ facility in Christchurch. In the ensuing following period, 31 of the mariners tested positive for COVID-19SARS-CoV-2 as did two workers at the MIQ facility (NZMH, 2020g). Genomeic sequencing produced full genomes for 24 cases and indicated at least four independent origins behind for the 33 cases, falling into three distinct phylogenetic clades (Fig. 1). The three clades consist oftain two, seven, and 15 cases respectively.

Here we have highlighted the outbreaks, and coloured them by their PANGO lineage.

We estimate the origin of the largest clade being between 4 August and 9 October (95% credible interval), but the mariners arrived on 16 October, suggesting there were at least two separate introductions of this variant into the facility. The two MIQ workers were infected with two different variants of the virus, with public health officials concluding that the two transmission events occurred via regular interactions with the mariners where protocols were followed but were insufficient to prevent transmission (Venter Consulting, 2020).

# [Sofrana Surville Cargo Vessel](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Sofrana%20Surville&label=internal:may18)

In mid October, a border worker tested positive for COVID-19 as part of routine testing. After extensive surveillance testing, three other case were identifieds tested positive; two were household contacts and one was a mariner who worked on board the same cargo vessel – the Sofrana Surville (NZMH, 2020h). Genomic testing confirmed that the four cases shared the same origin, and were part of a lineage that was novel to New Zealand, thus making residual community transmission an unlikely explanation. International crew members on board the Sofrana Surville were confirmed as the likely source of infection when the ship arrived in Australia and crew were tested by Australian health officials in Queensland. One crew member tested positive and the sequence was reported to match the New Zealand cases.

# [Defence Force Cluster](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Defence%20Force&label=internal:may18)

After visiting a number of public locations in the Auckland central business district, a New Zealand Defence Force member tested positive. Genomeic sequencing confirmed his the infection to be acquired from the quarantine facility he they worked at. Contact tracing identified three additional cases in Wellington, and surveillance testing identified one community case in the Auckland central business district. The only known connection to the rest of the outbreak was that this case worked at a retail outlet about 50 metres from one of the locations the index case visited (NZMH, 2020c). The case was quickly linked to the rest of the cluster through whole genome sequencing, providing reassurance that widespread undetected community transmission was unlikely (NZMH, 2020d). The circumstances of the transmission event remain unknown. Because the genomic link was established, there was no change in Alert Level. However, the general public were asked to avoid the Auckland central business district where possible for about the following three days.

# [Airline Crew cases 1 & 2](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Airline%20Crew%201,Airline%20Crew%202)

There have been two cases of an airline crew testing positive for COVID-19 after international travel.

In the first case, in December 2020, the crew member had returned from a flight from the United States of America. They were detected within 48 hours and there were no recorded secondary infections (NZMH, 2020a). Genomeic sequencing indicated they likely acquired their infection abroad.

The second case happened in March 2021 (during the Auckland Valentine’s outbreak). Here an airline crew member tested positive a week after their arrival from Japan (NZMH, 2021a). Their infection was likely acquired overseas and there were no known secondary infections were identified.


# [Pullman MIQ Facility](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Pullman&label=internal:apr04)

A returnee tested positive for COVID-19SARS-CoV-2 one week after completing managed isolation at the Pullman MIQ facility. Despite extensive travel across the Northland region whilst infected with the B.1.351 lineage (Tegally et al., 2020; Cele et al., 2021), there were no secondary infections reported, including their travelling companion. Shortly thereafter, two additional cases with the same variant were found in the community, both of whom had completed self-isolation at Pullman (NZMH, 2021c). The outbreak was successfully limited to these four community cases who were genomically linked to a returnee under in isolation at Pullman. The epidemiological link was thought to be the air circulation system at the MIQ facility. New arrivals at the MIQ facility were suspended and its air filtration systems were improved.

# [Auckland Valentine’s Lockdowns](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Auckland%20Valentine&label=outbreak:Auckland%20Valentine)

On the eve of Valentine’s day 2021, three household members in Auckland tested positive. The genomes were identical and of the highly transmissible B.1.1.7 variant (Volz et al., 2021; Davies et al., 2021). Although one case worked at an airline services company, they had no obvious contact with people coming through the border. Auckland was immediately sent into its third Alert Level 3 lockdown for a period of three days, and wide scale surveillance testing commenced. Other cases with closely related genomes were found, however the epidemiological links were not always strong (NZMH, 2021d). A fourth Alert Level 3 lockdown followed shortly afterwards when a further case was found that could not immediately be linked to the cluster.

The known outbreak was restricted to four households and 15 cases, although epidemiological gaps suggest there may have been undetected cases. With genetic evidence showing four distinct genomes among 14 cases, the genomics were not informative about the likely source beyond confirming a single origin, and the B.1.1.7 variant’s global overrepresentation presented further difficulties at pinpointing an overseas origin. The outbreak has not been traced back beyond the original three cases and its origin remains unknown.

# [Grand Millennium MIQ Facility](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Auckland%20Valentine,Grand%20Millennium&label=internal:sep26)
In late March 2021, a cleaner at the Grand Millennium MIQ facility tested positive. Despite being infected with the more transmissible B.1.1.7 lineage, there were no known secondary infections in the community. Two weeks later, two further workers from the same facility also tested positive (NZMH, 2021b). All three cases were genomically linked back to a returnee isolating at the Grand Millennium.

# [Auckland Airport](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?f_outbreak=Auckland%20Valentine,Grand%20Millennium,Auckland%20Airport&label=internal:sep26)

A worker at Auckland International Airport tested positive for the B.1.1.7 variant in April 2020. No onward transmission was detected. Genome sequencing linked the case to a recent returnee, and a followup investigation suggested that worker’s infection was obtained from cleaning the airplane the returnee had arrived on, and was likely a case of airborne transmission (NZMH, 2021e).


# [A wide range of lineages](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?c=lineage)

Zooming out and colouring the tree by lineage, we can see that a wide range of lineages have been imported into Aotearoa over the course of the pandemic (Geoghegan et al., 2020). The fact that the four locally acquired outbreaks in 2021 were all caused by variants of concern (VoC) – Pangolin lineages B.1.1.7 and B.1.351 (Wang et al., 2021a,b; Rambaut et al., 2020) – reflects the lineages that are arriving at the border, with 83 out of the 142 genomes from overseas returnees found from 1 January to 30 April 2021 being from those two lineages or other VoCs. However, there is too littleinsufficient data here to make any link between these outbreaks and the reported higher transmissibility of these VoCs.

# [Final words](http://localhost:4000/community/ESR-NZ/GenomicsNarrativeSARSCoV2/aotearoa-border-incursions?c=lineage)

New Zealand’s ability to rapidly generate SARS-CoV-2 genomes has greatly improved over the past yearduring the early months of the pandemic, to the point where new genomes are now routinely available within 24 hours of positive community tests. Although the potential of the techniques described here have been well characterised in academia (Gardy and Loman, 2018), the pandemic has facilitated led to widespread adoption in New Zealand and other places (such as Singapore and Australia (Pung et al., 2020; Seemann et al., 2020)); with the term “whole genome sequencing” becoming commonplace in public health announcements. Combined with epidemiological investigation, these data have increased public understanding of the outbreak and driven policy changes to minimise infection risks.

Although the techniques described here of real-time sequencing and analysis coupled with epidemiological investigation have come to the fore during the COVID-19 pandemic, they are not limited to pandemic situations. These technologies can be integrated into regular surveillance of other pathogens, such as seasonal influenza viruses, which have been largely absent from many countries over the past year (Olsen et al., 2020).

For more information please see the paper which this narrative is based on [here](https://www.medrxiv.org/content/10.1101/2021.05.13.21257194v1.full.pdf+html), or click "Explore the Data Yourself" (top right of screen) to interact with this tree.
