---
title: Re-emergence of community transmission in Aotearoa New Zealand - Genomic overview of the Auckland Outbreak
authors:
  - James Hadfield (†)
  - Jordan Douglas (†)
  - Jemma L Geoghegan
  - Matthew Storey
  - Xiaoyun Ren
  - Nikki Freed
  - Olin Silander
  - Cinthy L. Jiménez-Silva
  - Nigel French
  - Alexei J Drummond
  - David Welch
  - Joep de Ligt
authorLinks:
  - mailto:jhadfiel@fredhutch.org
  - 
  - https://micro.otago.ac.nz/our-people/teaching-research-and-support/jemma-geoghegan/
  -
  - 
  - https://www.massey.ac.nz/massey/expertise/profile.cfm?stref=844350
  - https://www.massey.ac.nz/massey/expertise/profile.cfm?stref=293350
  - 
  - https://www.massey.ac.nz/massey/expertise/profile.cfm?stref=219830
  - https://www.cs.auckland.ac.nz/people/a-drummond
  - https://unidirectory.auckland.ac.nz/profile/david-welch
  - https://scholar.google.com/citations?user=z2edTLkAAAAJ&hl=en
dataset: "https://nextstrain.org/community/ESR-NZ/GenomicsNarrativeSARSCoV2/2020-10-01?d=map&legend=closed"
abstract: "
This interactive situation report provides a high-level overview of the genomic data from the August 2020 re-emergence of SARS-CoV-2 in New Zealand (the 'Auckland August Cluster'). Before this outbreak was found, New Zealand had over 100 days without detecting community transmission.

_(†) these authors contributed equally_
"
---


# [Coronavirus in Aotearoa New Zealand prior to August 2020](https://nextstrain.org/community/ESR-NZ/GenomicsNarrativeSARSCoV2/2020-10-01?d=tree,map&f_region=New Zealand&legend=closed&dmax=2020-08-01)

Prior to 11 August 2020, New Zealand had over 1200 confirmed cases of COVID-19, spread throughout the country, and a further 350 probable cases.

The New Zealand Government has pursued an elimination strategy, which saw the imposition of a nation-wide lockdown on all non-essential services in late March. The lockdown was eased by late April and, from mid May, there followed a 103 day period with no community transmission detected.
To date the borders are open only to residents or those with an exemption. All arrivals are must stay in managed isolation and quarantine (MIQ) facilities for 14 days and are tested on days 3 and 12 for COVID-19.
Both tests tests must return a negative result prior to release from MIQ.

During the 103 days with no detected community transmission, there were a small but regular number of cases detected within MIQ facilites.


### Genome Sequencing and analysis

COVID-19 is the disease caused by a coronavirus known as SARS-CoV-2. 
Prior to August 2020 we sequenced over 700 SARS-CoV-2 genomes collected in New Zealand, shown in colour in the phylogenetic tree here. A number of worldwide isolates are included (in grey) for context.
The tree shows that New Zealand had multiple imported cases from around the world, some of which resulted in community transmission and spread in New Zealand.
In total, there were [16  "significant clusters" of community transmission, each containing 10 or more cases. The largest cluster had 98 cases.](https://www.health.govt.nz/our-work/diseases-and-conditions/covid-19-novel-coronavirus/covid-19-current-situation/covid-19-current-cases/covid-19-significant-clusters).



### More information

For more detailed genomic and phylogenetic analysis information please see:
* Jemma L. Geoghegan _et al._, Genomic epidemiology reveals transmission patterns and dynamics of SARS-CoV-2 in Aotearoa New Zealand, [medRxiv (2020)](https://www.medrxiv.org/content/10.1101/2020.08.05.20168930v3)
* Jordan Douglas and Fábio K. Mendes _et al._, Phylodynamics reveals the role of human travel and contact tracing in controlling COVID-19 in four island nations, [medRxiv (2020)](https://www.medrxiv.org/content/10.1101/2020.08.04.20168518v1)


# [Re-emergence of community cases in August 2020](https://nextstrain.org/community/ESR-NZ/GenomicsNarrativeSARSCoV2/2020-10-01?d=tree,map&f_region=New Zealand&legend=closed&dmin=2020-08-01&c=nz-group)

In August 2020, Aotearoa New Zealand saw the reappearance of COVID-19 in the community, with [four confirmed cases in one family](https://www.health.govt.nz/news-media/media-releases/4-cases-covid-19-unknown-source)
announced on the 11th of August. The source of this outbreak is still unknown.
To control the new outbreak, Auckland was immediately placed in [Alert level 3](https://covid19.govt.nz/covid-19/alert-system/auckland/#auckland-is-at-alert-level-3) with the regional borders closed, and the remainder of New Zealand moved to Alert Level 2.

By 9 September, genome sequencing confirmed that all sequenced positive cases (123) belong to a single cluster (here coloured red), with the exception of one case - a local maintenance worker at an MIQ facility, here coloured yellow - which was found to be linked to a recent case in MIQ.
In total, there were [171 cases](https://www.health.govt.nz/our-work/diseases-and-conditions/covid-19-novel-coronavirus/covid-19-current-situation/covid-19-current-cases/covid-19-significant-clusters) in the cluster as of 9 September 2020
and the [outbreak is thought to be contained](https://theconversation.com/aucklands-rapid-lockdown-has-given-new-zealand-a-better-chance-of-eliminating-coronavirus-again-145011).

Positive cases continue to be detected in MIQ facilities, and are sequenced as a matter of course (these are the blue isolates in the figure). As of 30 September, there have been three detected cases of onward transmission from MIQ facilities: the one maintenance worker mentioned above thought thought to have been infected [using an elevator shortly after a known case](https://www.stuff.co.nz/national/health/coronavirus/122512325/coronavirus-mystery-rydges-case-possibly-linked-to-use-of-hotel-elevator), a [healthcare worker](https://www.nzherald.co.nz/nz/news/article.cfm?c_id=1&objectid=12364506) from a quarantine facility, and a [returnee who tested positive](https://www.stuff.co.nz/national/health/coronavirus/122827196/coronavirus-returnees-should-selfisolate-after-managed-isolation-expert-says) after leaving managed isolation.


### This report

This report is a companion to the public release of the genomic data and summarises what we do and don't know about the genomic data up to 10 September 2020.


# [The Auckland cluster](https://nextstrain.org/community/ESR-NZ/GenomicsNarrativeSARSCoV2/2020-10-01?d=tree,map&legend=closed&label=nz:aac&m=div)

We first examine the SARS-CoV-2 sequences isolated from the ongoing community cluster (the "Auckland August cluster").
The tree here has been 'zoomed' and we are using a horizontal axis showing mutations, or observed genetic changes between the isolates -- you can hover over tips (circles) or branches (lines) to see the mutations and extra data on each isolate.

### All samples are genetically linked

We can see that all isolates form a single "clade" or grouping in the tree, with no worldwide isolates involved. This is to be expected for a cluster which has spread within New Zealand and originated from a single viral source.

### Genome sequencing has been able to inform contact tracing efforts

There have been a number of community cases which we were able to place in this cluster based on the genomic sequence where an epidemiological link was not yet established.
This demonstrates the power of genomics to aid our public health response to COVID-19, especially where data can be processed rapidly.

For more detailed information on how genomics has helped fill in the picture of this cluster, see the article ["Genome sequencing tells us the Auckland outbreak is a single cluster — except for one case"](https://theconversation.com/genome-sequencing-tells-us-the-auckland-outbreak-is-a-single-cluster-except-for-one-case-144721).


# [Where did the Auckland cluster come from?](https://nextstrain.org/community/ESR-NZ/GenomicsNarrativeSARSCoV2/2020-10-01?d=tree,map&legend=open&label=nz:aac-context&m=div&c=region&r=country)

We now zoom out to view genomes collected from around the world (including from the first wave in New Zealand) which are similar to sequences in the Auckland cluster.
We've now coloured the tree by worldwide region, with New Zealand isolates highlighted in pink.


The entire Auckland cluster comes from a part of the tree, or lineage, that [scientists refer to](https://www.biorxiv.org/content/10.1101/2020.04.17.046086v1) as [B.1.1.1](https://www.newshub.co.nz/home/new-zealand/2020/08/it-s-b-1-1-1-kiwi-scientists-identify-virus-family-of-mystery-covid-19-outbreak.html). 
The fact that all are classified as B.1.1.1 does not mean all the Auckland genomes are identical.
Every time the virus copies itself mistakes (or mutations) can occur.
These mutations leave a trail of ‘genetic breadcrumbs’ that speak to the recent history of the virus.

### Source unknown

We do not yet know the source of the Auckland cluster, but there are several hypotheses of how it arose. The most likely scenario is:
1. From a recent international traveller into New Zealand who picked up this viral lineage in another country. This lineage then evaded border measures put in place to contain COVID-19 in MIQ.

Two additional scenario's are also covered in this narrative:
2. From ongoing (undetected) transmission in New Zealand from the first wave of the epidemic.
3. As the result of so-called fomite transmission on packaging of imported fresh or frozen food.

The next three pages examine each hypothesis in more detail.

**Please be aware that we currently have no direct evidence for any of these scenarios.**
Indeed, we may never determine the root-cause, but we can explore the probability of these scenarios occurring.


### Date of origin

Our Bayesian analysis estimates of the genomes alone suggest that the first transmission event in the Auckland cluster was on 31 July 2020, with a 95% credible interval of (22 Jul till 09 Aug).

This matches well with the epidemiological evidence that the earliest known case [displayed symptoms of COVID-19 on 31 July](https://www.stuff.co.nz/business/industries/122446076/were-leaving-no-stone-unturned-did-new-zealands-outbreak-come-from-a-melbourne-coolstore).


# [Hypothesis 1: Border incursion from overseas](https://nextstrain.org/community/ESR-NZ/GenomicsNarrativeSARSCoV2/2020-10-01?c=region&d=tree,map&label=nz:aac-context&legend=open&m=div&r=country)

Very similar genomes to the Auckland lineage (1 nucleotide difference) have been observed in many countries and continents around the world.
It is therefore very plausible that an international arrival brought back the virus and subsequently transmitted it to the community.

While we have attempted to sequence all of the positive cases identified at the border, [around 40%](https://twitter.com/Joepdl/status/1295449744353845249) contain too little or too degraded genetic material (RNA) to obtain a full genome sequence.

There may be some unknown MIQ cases too. The swab test for COVID-19 is [known to miss a proportion of cases (false negatives)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7240870/), so even the MIQ testing regime (with tests on day 3 and 12) would be expected to miss [4% of true-positive cases](https://theconversation.com/genome-sequencing-tells-us-the-auckland-outbreak-is-a-single-cluster-except-for-one-case-144721).

There are other people crossing the border too, such as airline or shipping workers, who cannot be categorically ruled out as possible sources.

Finding a recent case in MIQ or elsewhere on the border that matched this cluster lineage would be strong evidence for the border incursion scenario, but the lack of such data does not rule it out as a posibillity.

### Was it the UK?

There are a large number of similar genomes which are from the UK, which would seem to suggest the UK is the most likely source of any unknown importation.

But we need to be cautious. Of the over 80,000 publicly available worldwide SARS-CoV-2 genomes, over 35,000 have been contributed by labs in the UK (an amazing feat!).
This sampling bias means that we would expect to see a higher number of UK B.1.1.1 isolates than from elsewhere simply by chance, making it hard to draw stronger conclusions.

### Was it Ecuador?
Two genomes from Ecuador (Ecuador/USFQ-194/2020 and Ecuador/USFQ-183/2020) were sampled in August 2020 and are only 1 mutation away from the Auckland cluster.

They had sister clade probabilities of over 60% in our most recent analysis (see next page).
However, as there were no returnees into New Zealand from the start of June until the end of July when the first Auckland cluster case developed symptoms, it is therefore very unlikely that the outbreak started from an Ecuador returnee through an MIQ facility.


# [Hypothesis 2: Undetected community transmission](https://nextstrain.org/community/ESR-NZ/GenomicsNarrativeSARSCoV2/2020-10-01)

There is one New Zealand isolate which is genetically similar (1 mutation difference) to the current Auckland cluster -- sample "20VR2563".
This sample comes from a case who was known to be infected before arrival in New Zealand and was immediately quarantined in Auckland. The sample was taken on April 14th 2020.

Our Bayesian phylogenetic analysis (on the right) estimates that there is a 0.4% probability that case 20VR2563 is in the ''sister clade'' of the Auckland cluster.
According to this analysis, it is much more likely that sequences in the sister clades come from Ecuador, South Africa, Spain, England, or Australia.

One reason the Auckland cluster does not sit closer to 20VR2563 in the tree is that although 20VR2563 is only one mutation away, it is **over three months** away in time.
Given what we know about the mutation rate of SARS-CoV-2 we would expect to have seen more mutations separating sequences over this period of time.
The virus mutates with an average of [2 mutations each month](https://www.who.int/bulletin/volumes/98/7/20-253591/en/).

Secondly, the genomic assembly of 20VR2563 was incomplete. The genome contains over 1000 positions which could not be resolved by genomic sequencing (i.e. missing sites). With these amounts of missing data, genomic inferences become weaker as these two genomes might not be as close as they appear.

There are also a number of epidemiological reasons that point to this hypothesis being unlikely.
Somehow the infection must have escaped from managed isolation, then remained undetected for four months in the community, before turning up as just this single lineage in this recent cluster.
Modelling and simulation shows that each of these events, particularly the last two, are very unlikely and taken together are highly improbable.


```auspiceMainDisplayMarkdown

## BEAST2 sister-clade analysis

<div class="rimage default">

<img width="75%" class="plot" src="https://raw.githubusercontent.com/ESR-NZ/GenomicsNarrativeSARSCoV2/main/Images/sisterclades.png" />

</div>

```


# [Hypothesis 3: Fomite transmission via imported goods](https://nextstrain.org/community/ESR-NZ/GenomicsNarrativeSARSCoV2/2020-10-01)


Extensive testing and contact tracing determined the earliest case found to date was [an Americold coolstore worker who first showed symptoms on July 31](https://www.stuff.co.nz/business/industries/122446076/were-leaving-no-stone-unturned-did-new-zealands-outbreak-come-from-a-melbourne-coolstore), and the initial spread of the cluster centred around this coolstore which imports frozen goods.

This led to the hypothesis that the virus may have been imported on packaging material, where it could have survived in low temperature conditions, and then gone on to infect a worker at the coolstore.
This hypothesis is was given further credence by the possible genomic link to Ecuador (discussed earlier), since [viral particles has been found in China on frozen shrimp from Ecuador](https://www.globaltimes.cn/content/1197537.shtml). However,  no shipments from Ecuador were received by the coolstore in question.

Our Bayesian phylogenetic analysis (right) shows that the estimated mutation rate on the branch leading to the cluster is not a lot smaller than elsewhere in the tree, lending little weight to the possibility that the virus lay dormant on packing material for a long period of time.


[Environmental testing of the coolstore facility](https://www.health.govt.nz/news-media/media-releases/5-new-cases-covid-19-3) on the 15th of August, including discarded packaging material, was carried out.
It found a small number of weak positive swabs in areas you would expect to find viral traces due to contamination from infected workers (e.g. in office areas and bathrooms).
Of several tested packing materials and areas, one weak positive sample was detected on packaging which was considered consistent with contamination.
It should be noted that swabbing of packaging over two weeks after the suspected date of first infection and that it is impractical to swab all potential surfaces.

Finally, no [cases of fomite transmission of SARS-CoV-2](https://www.who.int/news-room/commentaries/detail/transmission-of-sars-cov-2-implications-for-infection-prevention-precautions) have been confirmed internationally.
At the time of writing, a case identified in Christchurch MIQ is [suspected of being via fomite transmission](https://www.nzherald.co.nz/nz/news/article.cfm?c_id=1&objectid=12369825) with investigations ongoing. 

Confirming or excluding fomite transmission is a very hard prove in non-laboratory or un-controlled settings, but thus far it seems that fomite transmission is not very common.




```auspiceMainDisplayMarkdown


## Nucleotide substitution rates in the parental branch of the Auckland cluster

<div class="rimage default">
<img width="75%" class="plot" src="https://raw.githubusercontent.com/ESR-NZ/GenomicsNarrativeSARSCoV2/main/Images/branchrates.png" />
</div>


```

# [The Rydges hotel case](https://nextstrain.org/community/ESR-NZ/GenomicsNarrativeSARSCoV2/2020-10-01/rydges?legend=open&d=tree)

The "Rydges hotel case" was the first detected case whose genome did not match the lineage associated with the main Auckland cluster. [A worker](https://www.health.govt.nz/news-media/media-releases/results-covid-19-positive-cases-under-investigation-returned) at an Auckland isolation facility tested positive for COVID-19 with no known epidemiological links to the ongoing Auckland cluster and who did not have any routine contact with guests.


Genomic sequencing of the Rydges hotel case (NewZealand/AUK-ESR-20CV0071/2020, here coloured yellow) positioned it in a part of the tree dominated by samples from the USA that also included another NZ sample from July 1st (NewZealand/20VR3848/2020, not currently available on GISAID).
This was also from a returnee who previously tested positive in managed isolation and who was epidemiologically highlighted as having transitioned through the Rydges hotel.

These findings prompted further epidemiological investigation by health officials.
Data subsequently showed that the maintenance worker entered the lift [a matter of minutes](https://www.rnz.co.nz/national/programmes/checkpoint/audio/2018760401/covid-19-rydges-maintenance-man-used-same-lift-as-first-case) after the positive case left it. The current working theory is that the elevator, is the source of transmission between these two people.

This case highlights the utility of rapid genome sequencing in identifying the link and ruling out that this case was part of the Auckland cluster.


# [Things may change](https://nextstrain.org/community/ESR-NZ/GenomicsNarrativeSARSCoV2/2020-10-01?d=map&legend=closed)

The narrative presented here is based on analysis and knowledge at a single point in time. A final and salient point is that as we, and others, collect more genomics data and build knowledge of COVID-19 transmission pathways the evidence and assumptions may change.